### ISSUE TITLE
Not really an issue but a question

### ISSUE THREAD

**Issue by EeshaanJain** [mattpoggi/pydnet#18]
Hey, how could I get the depth values from the depth image using this? I need like 5metres etc.

**Comment by Christoph9402** [comment]
Hello, I am having the same question. Did you find an answer to that?

**Comment by EeshaanJain** [comment]
> Hello, I am having the same question. Did you find an answer to that?

@Christoph9402 No, I didn't, so I shifted to stereo-depth estimation for my project.
